src/app/src/sensorUtils.js

```diff
@@ -72,6 +73,12 @@ export function parseRiverGaugeApiData(id, data) {
                     [variable]: Number(sensorReading.value),
                     timestamp: new Date(sensorReading.dateTime),
                 });
+            } else {
+                // If sensor data is missing for a particular variable,
+                // use the default variable value for the sensor
+                return Object.assign(acc, {
+                    [variable]: DEFAULT_SENSOR_DATA[id][variable],
+                });
```

Comment:
There's opportunity in the below function `parseRiverGaugeCsvData` line 105 to use default values. It looks like we're defaulting to 0.

Reply:
Good catch. Updated.